pandas/core/arrays/arrow/array.py

```diff
@@ -1032,6 +1032,9 @@ def _reduce(self, name: str, *, skipna: bool = True, **kwargs):
             not_eq = pc.not_equal(data_to_cmp, 0)
             data_to_reduce = not_eq
 
+        elif name in ["min", "max", "sum"] and pa.types.is_duration(pa_type):
```

Comment:
Could you open up a pyarrow issue to see if they would support `min/max/sum` for duration types in the future?